packages/static-module-record/test/fixtures/large.js

```diff
@@ -570,17 +570,17 @@ function makeModulePlugins(options) {
             const vids = declarations.flatMap(({ id }) =>
               collectPatternIdentifiers(path, id),
             );
-            vids.forEach(({ name }) => {
+            for (const { name } of (vids)) {
               let tle = topLevelExported[name];
               if (!tle) {
                 tle = [];
                 topLevelExported[name] = tle;
               }
               tle.push(name);
-            });
+            }
           }
 
-          specs.forEach(spec => {
+          for (const spec of (specs)) {
```

Comment:
etc

Reply:
Using the regexp 
```
 of \((\w+)\)\) \{
```
I think the four you found were all of them. In any case, done.